pom.xml

```diff
@@ -13,6 +13,44 @@
     <artifactId>inamik-text-tables</artifactId>
     <version>0.8</version>
 
+    <properties>
```

Comment:
Definitely a good practice to use properties for versions and configurations.

👍